src/oge/load_data.py

```diff
@@ -116,21 +121,215 @@ def load_cems_data(year):
     return cems
 
 
-def load_cems_ids(start_year, end_year):
-    """Loads CEMS ids for multiple years."""
+def load_cems_ids() -> pd.DataFrame:
+    """
+    Loads a dataframe of all unique plant_id_eia, emissions_unit_id_epa combinations
+    that exist from teh earliest_data_year to the latest_validated_year. This is used
+    in the process of creating subplant_ids to ensure complete coverage.
+    """
 
-    # load cems data
-    cems = pd.read_parquet(
-        downloads_folder("pudl/hourly_emissions_epacems.parquet"),
-        filters=[["year", ">=", start_year], ["year", "<=", end_year]],
-        columns=["plant_id_epa", "plant_id_eia", "emissions_unit_id_epa"],
-    ).drop_duplicates()
-    cems = apply_pudl_dtypes(cems)
+    # although we could directly load all years at once from the cems parquet file, this
+    # would lead to a memoryerror, so we load one year at a time and drop duplicates before
+    # concatting the next year to the dataframe
+    cems_ids = []
+    # use 2001 as the start year as this is the earliest year that EIA data is available
```

Comment:
2001 -> 2005. Also, it would be worth mentioning that `earliest_data_year = 2005`

Reply:
Fixed